stytra/tests/test_req.txt

```diff
@@ -0,0 +1,39 @@
+pyqtgraph>=0.10.0
+numpy
+numba
+pandas
+qdarkstyle
+qimage2ndarray
+flammkuchen
+anytree
+pims
+GitPython
+colorspacious
+arrayqueues>=1.2.0b0
+pillow
+scikit-image
+imageio
+imageio-ffmpeg
+lightparam>=0.3.7
+opencv-python 
+pyFirmata 
+av 
+flammkuchen 
+gitpython 
+PIMS 
+qimage2ndarray 
+pyqtgraph 
+colorspacious 
+arrayqueues 
+anytree 
+qdarkstyle 
+psutil 
+pytest-qt
+pytest-forked
+pytest-xdist
+flake8
+tox
+Sphinx
+pytest
+pytest-cov
+black
```

Comment:
added requirement file for github workflow

Reply:
I don't know what @vilim thinks, but we should probably parse the requirements out of the `setup.py` and put them in a `requirements.txt`, so we can avoid this duplication here, maybe we can already fix this in this PR

Reply:
I've deleted it now, and added some requirements to requirements_dev.txt   and one to the setup.py (pyFirmata